appengine/py/appengine_deploy.sh.template

```diff
@@ -19,37 +19,34 @@ case "$0" in
 esac
 if [[ -e "$self.runfiles/%{workspace_name}" ]]; then
   RUNFILES="$self.runfiles/%{workspace_name}"
-  cd $RUNFILES
+  cd "$RUNFILES"
 fi
 
-ROOT=$PWD
-tmp_dir=$(mktemp -d ${{TMPDIR:-/tmp}}/war.XXXXXXXX)
-cp -R $ROOT $tmp_dir
-trap "{{ cd ${{root_path}}; rm -rf $tmp_dir; }}" EXIT
-rm -Rf $tmp_dir/%{workspace_name}/external/com_google_appengine_py
-if [ -n "${{1-}}" ]; then
-  has_app_yaml=$(echo ${{@:2}} | grep -E "^([^ ]+ +)*app.yaml")
-  other_mod_cfgs=$(echo ${{@:2}} | xargs printf "%s\n" | grep -v "^app\.yaml$" | xargs echo)
+ROOT="$PWD"
+tmp_dir="$(mktemp -d "${TMPDIR:-/tmp}/war.XXXXXXXX")"
+cp -R "$ROOT" "$tmp_dir"
+trap "{ cd "$ROOT"; rm -rf "$tmp_dir"; }}" EXIT
+rm -Rf "$tmp_dir/%{workspace_name}/external/com_google_appengine_py"
+if [ -n "${1-}" ]; then
+  has_app_yaml="$(echo "${@:2}" | grep -E "^([^ ]+ +)*app.yaml")"
+  other_mod_cfgs="$(echo "${@:2}" | xargs printf "%s\n" | grep -v "^app\.yaml$" | xargs echo)"
```

Comment:
optional: Some of this code is brittle against paths with spaces in them, but users probably shouldn't be using those anyway. If you really care, there's probably a way to do all this in just bash. Maybe something like:

```shell
has_app_yaml=false
other_mod_cfgs=()

for p in "${@:2}"; do
  if [[ "$p" == "app.yaml" ]]; then
    has_app_yaml=true
  else
    modules+=("$p")
  fi
done
```

And then you can use `${other_mod_cfgs[@]}` to pass all of the other `.yaml` files to appcfg update below.